test/e2e/serial/workload_unschedulable_test.go

```diff
@@ -196,7 +196,7 @@ var _ = Describe("[serial][disruptive][scheduler] workload unschedulable", func(
 			dsName := "test-ds"
 
 			podLabels := map[string]string{
-				"test": "test-dp",
+				"test": "test-daemonset",
```

Comment:
That's correct. Here we're in the `It` block concerning a daemonset (see lines 194-195), so we're fixing a typo, should have been `test-ds` since the beginning. I'm not sure we should highlight more the typo fix.